pkg/zonemgr/zone_file.go

```diff
@@ -2,180 +2,151 @@ package zonemgr
 
 import (
 	"fmt"
-	v1 "kubevirt.io/api/core/v1"
+	"k8s.io/utils/net"
+	"sort"
 	"strconv"
+
+	v1 "kubevirt.io/api/core/v1"
 )
 
 const (
 	refresh = "3600"    // 1 hour (seconds) - how long a nameserver should wait prior to checking for a Serial Number increase within the primary zone file
 	retry   = "3600"    // 1 hour (seconds) - how long a nameserver should wait prior to retrying to update a zone after a failed attempt.
 	expire  = "1209600" // 2 weeks (seconds) - how long a nameserver should wait prior to considering data from a secondary zone invalid and stop answering queries for that zone
 	ttl     = "3600"    // 1 hour (seconds) - the duration that the record may be cached by any resolver
+
+	domainDefault     = "vm"
+	nameServerDefault = "ns"
+	adminEmailDefault = "email"
```

Comment:
aint it need to be some a@b.com format?
well if it works, i am fine with just the word email as you did

Reply:
the format can be domain based, for example: admin.example.com
"the administrator's email address in an SOA record admin.example.com is the equivalent of admin@example.com."